src/uniswap-summaries.md

```diff
@@ -3155,6 +3155,154 @@ module SOLIDITY-UNISWAP-SWAP-SUMMARY
        <current-function> swap </current-function>
     requires Vto =/=MInt {S[token0] orDefault default(address)}:>MInt{160} andBool Vto =/=MInt {S[token1] orDefault default(address)}:>MInt{160} [priority(40)]
 
+  // End of first if statement to second if condition evaluation
```

Comment:
Won't we summarize the first if condition?
How about the false case of the second if condition?

Reply:
Oh, it looks like I may have missed the commit with the opposite cases when rebasing. Let me take a look, and redo those. Thanks!

Reply:
I now handle the (missing before) true branch of the first if statement. I did not add anything for the cases where they evaluate to false, because we already have this rule in the semantics.
```
rule if ( v(false, bool ) ) _ => .K
```